src/java/org/apache/cassandra/index/sai/plan/StorageAttachedIndexSearcher.java

```diff
@@ -845,13 +864,12 @@ protected Unfiltered computeNext()
         }
     }
 
-    private static UnfilteredRowIterator applyIndexFilter(UnfilteredRowIterator partition, FilterTree tree, QueryContext queryContext)
+    private static UnfilteredRowIterator applyIndexFilter(UnfilteredRowIterator partition, FilterTree tree)
     {
-        FilteringPartitionIterator filtered = new FilteringPartitionIterator(partition, tree, queryContext);
+        FilteringPartitionIterator filtered = new FilteringPartitionIterator(partition, tree);
         if (!filtered.hasNext() && !filtered.matchesStaticRow())
         {
             // shadowed by expired TTL or row tombstone or range tombstone
```

Comment:
Shouldn't we remove also the comment here?

Reply:
I left the comment because it still applies to the rest of the block. But on second thought probably the intention was about the metric, and I think there might some unrelated issues in this block that I'd like to investigate in a separate issue, so I'm removing it.